app/javascript/react/components/SauceShowPage.js

```diff
@@ -28,7 +29,7 @@ const SauceShowPage = (props) => {
       return null
     }
      
-    const reviewTiles = sauce.reviews.map((review)=>{
+    const reviewTiles = sauce.reviews.reverse().map((review)=>{
```

Comment:
you shouldnt use reverse. If you want the order of the reviews to be different, then you should retrieve them differently on the backend

Reply:
Removed reverse